synapse/lib/stormtypes.py

```diff
@@ -5283,111 +5310,42 @@ def __rmod__(self, othr):
     async def stormrepr(self):
         return str(self.value())
 
-@registry.registerLib
-class LibGlobals(Lib):
+@registry.registerType
+class GlobalVars(Prim):
```

Comment:
Maybe a `stormrepr()` on this class? Could probably copy the stormrepr method from Dict.

Reply:
added